ArchaicQuestII.GameLogic/Commands/Character/DismountCmd.cs

```diff
@@ -38,7 +38,7 @@ public void Execute(Player player, Room room, string[] input)
         {
             if (string.IsNullOrEmpty(player.Mounted.Name))
             {
-                Services.Instance.Writer.WriteLine("<p>You are not using a mount</p>");
+                Services.Instance.Writer.WriteLineAll("<p>You are not using a mount</p>");
```

Comment:
These look like old mistakes, there are 5 others but they shouldn't be emoting to all, should be using WriteLine and passing the player object in

Reply:
Took care of these